ui/lib/core/addon/templates/components/replication-mode-summary.hbs

```diff
@@ -104,10 +104,8 @@
       {{#link-to "vault.cluster.replication.mode.index" cluster.name mode class="button is-primary"}}
         Enable
       {{/link-to}}
-    {{else if (eq mode 'dr')}}
-      {{cluster.drReplicationStateDisplay}}
-    {{else if (eq mode 'performance')}}
-      {{cluster.perfReplicationStateDisplay}}
+    {{else}}
+      {{get (cluster-states modeState) "display"}}
```

Comment:
i haven't been able to figure out where this code is actually executed -- if anyone has any ideas please lemme know! otherwise, if i can get rid of this line it means i can get rid of all of the 'display' properties inside `CLUSTER_STATES`